drake/util/TrigPoly.h

```diff
@@ -3,8 +3,30 @@
 
 #include "drake/util/Polynomial.h"
 #include <map>
-//#include <iostream>  // just for debugging
 
+/// A scalar multi-variate polynomial containing sines and cosines.
+/**
+ * TrigPoly represents a Polynomial some of whose variables actually represent
+ * the sines or cosines of other variables.  Sines and cosines of first-order
+ * polynomials (affine combinations of variables) are decomposed into
+ * polynomials of the sines and cosines of individual variables via the
+ * Prosthaphaeresis formulae.
+ *
+ * Any variables which will appear in the arguments to trigonometric functions
+ * must be declared in the "SinCosMap"; attempting to, e.g., use sin(x)
```

Comment:
not clear here that "SinCosMap" is created by TrigPoly -- from the comment I was expecting to see "SinCosMap" in the code sample